src/vs/editor/contrib/folding/folding.ts

```diff
@@ -909,8 +909,8 @@ registerThemingParticipant((theme, collector) => {
 	const editorFoldColor = theme.getColor(editorFoldForeground);
 	if (editorFoldColor) {
 		collector.addRule(`
-		.monaco-editor .cldr.codicon-chevron-right,
-		.monaco-editor .cldr.codicon-chevron-down {
+		.monaco-editor .cldr.codicon-folding-collapsed,
+		.monaco-editor .cldr.codicon-folding-expanded {
```

Comment:
Another way would rely on `cssSelector` properties of https://github.com/microsoft/vscode/blob/d179acdbd04f84e9ac2fc96a68c2690457c5f77a/src/vs/editor/contrib/folding/foldingDecorations.ts#L12-L13

Similar approach was used in https://github.com/microsoft/vscode/blob/d179acdbd04f84e9ac2fc96a68c2690457c5f77a/src/vs/editor/contrib/documentSymbols/outlineTree.ts#L559-L567

/cc @aeschli

Reply:
Thanks for the tip, that makes sense -> refactored! 🍻

Reply:
TBH I was thinking about exporting these registered icons (or their selectors) and using them because `folding.ts` already imports from `foldingDecorations.ts`, instead of removing specific icons.

But @aeschli can weight in what approach is better.